Document nimbus-fml info command (#535)
mozilla/experimenter-docs: docs/deep-dives/specifications/fml/fml-cli.mdx

```
% nimbus-fml single-file --channel release @mozilla-mobile/firefox-android/fenix/app/nimbus.fml.yaml single-file.fml.yaml
```

## Generating a machine readable overview of a feature

```sh
nimbus-fml info [--channel CHANNEL] [--json] <MANIFEST> [--feature FEATURE]
```

This prints a simplified YAML or JSON representation for each feature. e.g.

```json
{
  "file": "https://raw.githubusercontent.com/mozilla-mobile/firefox-android/main/fenix/app/nimbus.fml.yaml",
  "features": {
    "homescreen": {
      "description": "The homescreen that the user goes to when they press home or new tab.",
      "types": [
        "Boolean",
        "HomeScreenSection",
        "Map<HomeScreenSection, Boolean>"
      ],
      "hashes": {
        "schema": "7a15570b",
        "defaults": "423ab0bd"
      }
    }
  }
}
```

> The `hashes` object shows truncated SHA256 hashes for:

Reviewer: Nit: add subheadings

```suggestion
### Hashes

The `hashes` object shows truncated SHA256 hashes for:
```

Reviewer: I think adding these would make the heading previews nicer:
<img width="443" alt="image" src="https://github.com/mozilla/experimenter-docs/assets/43795363/71a29b2e-ed73-4474-8fec-0d872533a342">

- the schema (the types, variable names, the enum variants available)
- the defaults (the default values of the feature variables)

Changes in the schema hash might indicate a change in the feature code.
Changes in the defaults hash would indicate a different configuration being used for the same code.

> :::info
> Changes in the schema hash will almost always be accompanied by a change in the defaults hash, with the exception of changing variable types from
> a `String` type to a `string-alias` type (or vice-versa).
> :::

Reviewer: ```suggestion
:::tip
Changes in the schema hash will almost always be accompanied by a change in the defaults hash, with the exception of changing variable types from
a `String` type to a `string-alias` type (or vice-versa).
:::

I almost want the two above "Changes in..." sentences to be in this tip box too 🤔 This all seems worth highlighting. Not sure though.
```

> The `types` list is a sorted list of the types involved used to define the feature. This list may be used as a proxy for a measure of complexity of the feature configuration space.

Reviewer: ```suggestion
### Types

The `types` list is a sorted list of the types involved used to define the feature. This list may be used as a proxy for a measure of complexity of the feature configuration space.
```

> In addition, [the feature metadata](/fml/feature-metadata) is also displayed.

Reviewer: ```suggestion
### Feature metadata

In addition, [the feature metadata](/fml/feature-metadata) is also displayed.
```
